realm-transformer/src/main/java/io/realm/transformer/RealmAnalytics.java

```diff
@@ -68,6 +68,7 @@
             + "      \"Anonymized MAC Address\": \"%USER_ID%\",\n"
             + "      \"Anonymized Bundle ID\": \"%APP_ID%\",\n"
             + "      \"Binding\": \"java\",\n"
+            + "      \"Uses Kotlin\": \"%USES_KOTLIN%\",\n"
```

Comment:
- It’s worth checking this will work as intended via a quick test but I think it might be better to return this under the `Language` key as a [MixPanel list value](https://mixpanel.com/docs/properties-or-segments/property-data-types).
- Values should be lowercase abbreviated forms of the language name if there are any. The current values other bindings return are `objc`, `swift`, `js` and `c#`. Unless there are any abbreviation I’m not aware of, just report `kotlin`, `java`, `groovy`, etc. ?
- That means we should also include `Java` as a value in that list if we have any reason to suspect Java is being used. (Right now since we technically do not know, all Java runs are reported with the `Language` property left undefined, which is correct since we do not know for sure)
- Note `Language` singular instead of plural, because other bindings already return that property key singular and we should be consistent to simplify reporting. Ideally that will let us do queries in Mixpanel like `Language is one of “java”, “groovy” or “js”.
- When in doubt, feel free to [go in MixPanel and check if language are reported as you expect](https://mixpanel.com/report/686603/segmentation/#action:segment,arb_event:Run,bool_op:and,chart_analysis_type:linear,chart_type:line,from_date:-56,ms_checked:(c%23:!t,js:!t,objc:!t,swift:!t,undefined:!t),ms_values:!(undefined,swift,objc,js,c%23),segfilter:!((dropdown_tab_index:0,property:(name:Language,source:properties,type:string),selected_property_type:string,type:string)),segment_type:string,to_date:-8,type:unique,unit:week)